ios/RNIapIosSk2.swift

```diff
@@ -277,10 +277,6 @@ class RNIapIosSk2: RCTEventEmitter, Sk2Delegate {
         delegate.stopObserving()
     }
 
-    override func addListener(_ eventName: String?) {
```

Comment:
code review note: 
- not my intention
- this was removed by `yarn swift:lint`, otherwise the lint is failing